modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/BaseDDMTestCase.java

```diff
@@ -137,12 +138,17 @@ public static void setUpClass() {
 
 		_ddmFormDeserializerServiceRegistration = bundleContext.registerService(
 			DDMFormDeserializer.class, _ddmFormDeserializer, properties);
+
+		_siteConnectedGroupGroupProviderServiceRegistration =
+			bundleContext.registerService(
+				SiteConnectedGroupGroupProvider.class,
+				Mockito.mock(SiteConnectedGroupGroupProvider.class), null);
 	}
 
 	@AfterClass
 	public static void tearDownClass() {
 		_ddmFormDeserializerServiceRegistration.unregister();
-
+		_siteConnectedGroupGroupProviderServiceRegistration.unregister();
```

Comment:
@JanisZhang I think we'd better add an empty line after this line, if you want to keep the mockedStatic.close last one in the method. Or Brian may ask you to sort in alphabet order...